.github/workflows/test-hadoop.yaml

```diff
@@ -0,0 +1,132 @@
+name: Run hadoop integration tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      branches:
+        description: 'Comma-separated list of branches to test'
+        required: true
+        default: 'lp-3.3.6,lp-3.3.6,lp-3.3.6'
```

Comment:
question, why do we have all three entries the same?

Reply:
It is an example to show that you can run test on a branch multiple times